.github/workflows/package-and-release.yml

```diff
@@ -0,0 +1,57 @@
+name: Create and Sign Package
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+env:
+  REPO: ${{github.event.repository.name}}
+  # Credentials for Jfrog artifactory
+  JF_URL: ${{ secrets.JF_URL }}
+  JF_ACCESS_TOKEN: ${{ secrets.ARTIFACTORY_TOKEN}}
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
```

Comment:
Is the `runs-on` needed since this also specifies the custom container to use?

Reply:
Ahh, I found this post explaining the difference: https://github.com/orgs/community/discussions/25534 Looks like this is good to go.